build-sensor/platformio.ini

```
@@ -0,0 +1,24 @@
+; PlatformIO Project Configuration File
+;
+;   Build options: build flags, source filter
+;   Upload options: custom upload port, speed and extra flags
+;   Library options: dependencies, extra library storages
+;   Advanced options: extra scripting
+;
+; Please visit documentation for the other options and examples
+; https://docs.platformio.org/page/projectconf.html
+
+[env:firebeetle32]
+platform = espressif32
+board = firebeetle32
```

This is for the beetle, should change for the class.
```suggestion
board = esp32dev
```